PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,2 +1,14 @@
--
--
+## What Jira issue is your PR related to?
+_e.g. [TBL-123](https://studyportals.atlassian.net/browse/TBL-123)_
```

Comment:
Sorry, late to the party, but is it a good idea to have this URL in a public repo?

Reply:
There is no access to it externally, so should be fine. But good that you paid attention!

Reply:
I don't think it's too much of a problem to leak the issue ID, as even if somebody gains access to Atlassian's database they still would be able to do much with the information they have from JIRA.